### PR TITLE
Add pre-filled search on package page

### PR DESCRIPTION
**Pull request by schnellecom**

### CODE REVIEW

**Comment by fingolfin** [comment]
As noted on issue #355 this needs to be adjusted to perhaps remove the `?q=` handling and/or replace the `#` code by using the `?q=` instead.

**Review by fingolfin — CHANGES_REQUESTED** [review]
It seems your dark mode changes crept in here as well? I would recommend developing them on a separate branch (happy to help you with that as well)

File: _config.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ url: "https://dev.gap-system.org" # the base hostname & protocol for your site,
 
 docsurl: "https://docs.gap-system.org"
 
-color_scheme: gap
+color_scheme: lightgap
```
[the remark this answers is not in the document]
Comment (schnellecom) [reply _config.yml]: No, my bad habit of doing `git add -A` has caught up with me. There should be reverts now, and a second branch in my fork that does contain the dark mode version.

File: .htaccess [hunk 2]
```diff
@@ -49,8 +49,7 @@ Redirect temp /Overview/Capabilities/representations.html /capabilities/represen
 Redirect temp /Overview/Capabilities/semigroups.html /capabilities/semigroups
 
 Redirect temp /Packages/Authors/authors.html /packages/authors
-Redirect temp /Packages/Authors/pkgauthorhints.html /packages/authors
-Redirect temp /Packages/Authors/variablenames.html /packages/authors/variablenames
+Redirect temp /Packages /packages
```
[the remark this answers is not in the document]
Comment (schnellecom) [reply .htaccess]: Done now

**Review by fingolfin — APPROVED** [review]

**Pull request merged**